### PR TITLE
[ADD] airflow scheduler plugin

### PR DESCRIPTION
**Pull request by prakshalj0512**

### CODE REVIEW

File: airflow-scheduler/airflow_scheduler.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+airflow_host=${1:-localhost}
+airflow_port=${2:-8080}
+
+scheduler_status=$(curl -k https://"${airflow_host}":"${airflow_port}"/health | jq --raw-output '.scheduler.status')
```
Comment (rssanders3) [inline airflow-scheduler/airflow_scheduler.sh]: Does this `jq` command come with Linux by default?

Comment (prakshalj0512) [reply airflow-scheduler/airflow_scheduler.sh]: It does not. I've mentioned it as a pre-req in the README.

File: airflow-scheduler/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+- The following guide demonstrates how to install a plugin to monitor the Airflow Scheduler through Nagios, an application that monitors systems, networks and infrastructure.
+
+- The plugin returns a CRITICAL state when the scheduler health is "unhealthy" and OK state when the health is "healthy." It returns UNKNOWN state when the API call cannot be made.
+
+## Deployment and Configuration
+
+### Prerequisites
+
+- The process should be running on a machine with the `curl` and `jq` packages installed.
```
Comment (rssanders3) [inline airflow-scheduler/README.md:13]: Mention how these packages are installed (yum, apt-get, download binaries, etc)

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: airflow-scheduler/README.md (hunk 2 above)
Comment (prakshalj0512) [reply airflow-scheduler/README.md:13]: Added

File: airflow-scheduler/airflow_scheduler.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+airflow_host=${1:-localhost}
+airflow_port=${2:-8080}
+
+scheduler_status=$(curl -k http://"${airflow_host}":"${airflow_port}"/health 2>/dev/null | jq --raw-output '.scheduler.status')
```
Comment (rssanders3) [inline airflow-scheduler/airflow_scheduler.sh]: Provide argument for http or https

File: airflow-scheduler/airflow_scheduler.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+echo $scheduler_status
```
Comment (rssanders3) [inline airflow-scheduler/airflow_scheduler.sh]: Provide more of a log then this:
echo "Scheduler Status: ${scheduler_status}"

**Review by rssanders3 — CHANGES_REQUESTED** [review]

File: airflow-scheduler/README.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+define service{
+
+    use local-service
+    host_name localhost
+
+    # service_description Airflow Scheduler <airflow_host>:<airflow_port>
+    service_description Airflow Scheduler http://localhost:8080
+
+    # check_command airflow_scheduler!<airflow_host>!<airflow_port>
+    check_command http!airflow_scheduler!localhost!8080
```
Comment (rssanders3) [inline airflow-scheduler/README.md]: Correct me if im wrong, but shoudlnt this be:
`check_command airflow_scheduler!http!localhost!8080`

Comments above should also be updated

**Review by rssanders3 — APPROVED** [review]
LGTM

**Pull request merged**